frontend/packages/network-attachment-definition-plugin/src/constants/index.ts

```diff
@@ -57,6 +68,21 @@ export const networkTypeParams: NetworkTypeParamsList = {
       initValue: true,
     },
   },
+  [ovnKubernetesSecondaryLocalnet]: {
+    bridgeMapping: {
+      name: 'Bridge Mapping',
```

Comment:
These user-facing strings should be sentence case, not title case. It would be great if you could correct the others in this file as well.

Also, according to the Jira ticket the "Bridge mapping" label should have a tooltip explaining what it is. I looked for a design document to see if a tooltip was included, but didn't see a doc linked in the ticket.

Reply:
Unfortunately, there isn't any design doc (this feature is not planned and prepared well, unfortunately, I asked for that but was told that it is not needed, I didn't agree but had no power to change that), there isn't any specific suggestion for the tooltip text (I asked for that but it seems nobody cares so probably it is not important anymore, so I wasn't trying to add it without any clear info because it doesn't worth the effort most of the time, it can be added later separately, when it will be specified and really needed) + I just followed the suggestions in the ticket, so I didn't use sentence case. The ticket says clearly about what they want to see in the UI. 

Anyway, the whole NADs section in the UI is very inconsistent with the rest of the UI, I am not going to fix this problem in this PR. If they will want it to be consistent, then I will use sentence case + will change all other related inconsistent texts, too. However, it seems nobody cares about consistency of NADs pages much, at least for now, except the two of us.
 
To sum up, I am not going to make these changes in this PR from various reasons, at least for now.

Reply:
I talked to Petr. He updated the casing to "Bridge mapping" and added the copy for the tooltip to the ticket. I asked Yifat and Avital to review it.

If you can update the case on "Bridge mapping" and add the tooltip once they finalize the copy I'll open a separate PR to update the remainder of the customer-facing strings since I'm sure I'm responsible for much of the inconsistency.

Reply:
Thanks. I'd wait for Avital to review it. She was pinged already longer time ago regarding this topic, but..